src/main/java/com/github/wovnio/wovnjava/Interceptor.java

```diff
@@ -1,608 +1,43 @@
-                content.setNodeValue(newContent);
-            }
-        }
-
-        NodeList imgs = null;
-        try {
-            imgs = (NodeList)xpath.evaluate("//*[local-name()='img']", doc, XPathConstants.NODESET);
-        } catch (XPathExpressionException e) {
-            // No erorr occurs.
-        }
-        for (int i = 0; i < imgs.getLength(); i++) {
-            Node img = imgs.item(i);
-            if (img == null || this.checkWovnIgnore(img)) {
-                continue;
-            }
-            Element imgElement = (Element)img;
-            String src = imgElement.getAttribute("src");
-            if (!Pattern.compile("://").matcher(src).find()) {
-                if (Pattern.compile("^/").matcher(src).find()) {
-                    src = url.get("protocol") + "://" + url.get("host") + src;
-                } else {
-                    src = url.get("protocol") + "://" + url.get("host") + url.get("path") + src;
-                }
-            }
-            String destSrc = values.getImage(src, lang);
-            if (destSrc != null) {
-                img.getAttributes().getNamedItem("src").setNodeValue(destSrc);
-            }
-
-            if (img.getAttributes().getNamedItem("alt") != null) {
-                String alt = img.getAttributes().getNamedItem("alt").getNodeValue();
-                alt = Pattern.compile("^\\s+|\\s+$").matcher(alt).replaceAll("");
-                String destAlt = values.getText(alt, lang);
-                if (destAlt != null) {
-                    String newAlt = Pattern.compile("^(\\s*)[\\S\\s]*(\\s*)$")
-                            .matcher(alt)
-                            .replaceAll("$1" + destAlt + "$2");
-                    img.getAttributes().getNamedItem("alt").setNodeValue(newAlt);
-                }
-            }
-        }
-
-        NodeList scripts = null;
-        try {
-            scripts = (NodeList)xpath.evaluate("//*[local-name()='script']", doc, XPathConstants.NODESET);
-        } catch (XPathExpressionException e) {
-            // No error occurs.
-        }
-        for (int i = 0; i < scripts.getLength(); i++) {
-            Element src = (Element)scripts.item(i);
-            if (src != null
-                    && Pattern.compile("//j\\.(dev-)?wovn\\.io(:3000)?/").matcher(src.getAttribute("src")).find()) {
-                src.getParentNode().removeChild(src);
-            }
-        }
-
-        Node parentNode = null;
+	private String apiTranslate(String lang, String body) {
         try {
-            NodeList heads = doc.getElementsByTagName("head");
-            if (heads != null && heads.getLength() > 0) {
-                parentNode = heads.item(0);
-            }
-        } catch (NullPointerException e) {
-            Logger.log.error("NullPointerException while searching <head> tag", e);
+            HtmlConverter converter = new HtmlConverter(settings, body);
+		    String convertedBody = converter.strip();
```

Comment:
Indent. I think you have mixed spaces and tab here

Reply:
You are right.
I fixed it.